examples/Client/DistributedLock/README.md

```diff
@@ -80,5 +80,5 @@ Note that both apps succeed and fail to lock files. This is due to the two apps
 Any files that are leftover can be safely removed.
 
 ```bash
-rm -rf ./tmp
+rm -rf /tmp/localstorage
```

Comment:
So my concern with this is if someone messes up the command, they could delete their actual `/tmp`. What happens when you use `./tmp`? Can you give an example of the actual error message?

Reply:
```
FATA[0000] process component localstorage error: [INIT_COMPONENT_FAILURE]: initialization error occurred for localstorage (bindings.localstorage/v1): failed to parse metadata: property rootPath must be an absolute path  app_id=generator instance=Yashs-MacBook-Pro.local scope=dapr.runtime type=log ver=edge
❌  The daprd process exited with error code: exit status 1

```

Reply:
I think we should just add a warning in the readme for warning the users.